src/dmd/declaration.d

```diff
@@ -541,6 +541,11 @@ extern (C++) abstract class Declaration : Dsymbol
         return (storage_class & STC.ref_) != 0;
     }
 
+    final bool isRefOrOut() const pure nothrow @nogc @safe
```

Comment:
What's the advantage of this method compared to calling `isRef || isOut`? We cannot have methods for any possible combination of conditions.

Reply:
Yes and the objective is not to represent all of them but the `ref` or `out` combination is commonly used when the semantic for arguments is done.